portfolio/src/main/webapp/script.js

```diff
@@ -143,4 +144,23 @@ function changeButtonValDown() {
         sessionStorage["current-page"]--;
     }
     setCurrentPage();
+}
+
+function login() {
+    fetch("/home")
+    .then(response => response.text())
+    .then(text => {
+        const commentSection = document.getElementById("add-comment");
+        console.log(text)
+        const parsedText = JSON.parse(text);
+        const link = document.createElement("a");
+        link.setAttribute("href", parsedText.url);
+        if (parsedText.isLoggedIn) {
+            commentSection.removeAttribute("hidden");
+            link.innerText = "Logout here."
+        } else {
+            link.innerText = "Login here to comment."
```

Comment:
maybe you should add the hidden attribute here just in case the user goes from logged in to logged out so the comments disappear.

Reply:
done!